### PR TITLE
Fixed last 10 lint warnings.

### PR DESCRIPTION
**Pull request by bhamodi**
@jordangarcia - this PR fixes the last lint warnings we have. :+1:

### CODE REVIEW

File: grunt/sauce.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+var each = require('../src/utils').each
```
Comment (jordangarcia) [inline grunt/sauce.js]: since this is part of the tooling you can just use lodash here.

File: tests/evaluator-tests.js [hunk 2]
```diff
@@ -168,8 +168,8 @@ describe('Evaluator', () => {
       var currentProjectGetter = [
         ['projects'],
         ['session', 'currentProjectId'],
-        (projects, currentProjectId) => {
```
Comment (jordangarcia) [inline tests/evaluator-tests.js]: Im not really a fan of this rule, we dont write a lot of large JS files with nested closures and sometimes using the same variable name is more correct than a "new" one.

**Comment by jordangarcia** [comment]
this lgtm

File: tests/evaluator-tests.js (hunk 2 above)
Comment (bhamodi) [inline tests/evaluator-tests.js]: We can change rules and fix the respective lint issues in future PRs as we change our minds about styling rules. :+1:

**Pull request merged**